src/vs/workbench/parts/quickopen/browser/gotoLineHandler.ts

```diff
@@ -91,12 +91,14 @@ class GotoLineEntry extends EditorQuickOpenEntry {
 
 		// Inform user about valid range if input is invalid
 		const maxLineNumber = this.getMaxLineNumber();
-		if (this.invalidRange(maxLineNumber)) {
+		const line = this.editorService.activeTextEditorWidget.getPosition().lineNumber;
+
+		if (this.invalidRange(maxLineNumber) && typeof line === 'number') {
 			if (maxLineNumber > 0) {
-				return nls.localize('gotoLineLabelEmptyWithLimit', "Type a line number between 1 and {0} to navigate to", maxLineNumber);
+				return nls.localize('gotoLineLabelEmptyWithLimit', "Current Line: {0}.  Type a line number between 1 and {1} to navigate to", line, maxLineNumber);
```

Comment:
@cleidigh there is an extra whitespace in the localize value

Reply:
👍